mender-auth/main.cpp

```diff
@@ -12,10 +12,32 @@
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
 
-#include <memory>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <common/conf.hpp>
+#include <common/context.hpp>
 
 using namespace std;
 
-int main() {
+int main(int argc, char *argv[]) {
+	mender::common::conf::MenderConfig config;
+	if (argc > 1) {
+		vector<string> args(argv + 1, argv + argc);
+		auto err = config.ProcessCmdlineArgs(args);
+		if (err) {
+			cerr << "Failed to process command line options: " + err.message << endl;
+			return 1;
+		}
+	}
```

Comment:
I realized (while driving a car home) we need to call something like `config.LoadDefaults()` here in an `else` branch. Will do it.